Automattic/woocommerce-payments
# Move from docker-based to standalone Jurassic Tube setup

Related to https://github.com/Automattic/transact-gateway/pull/460

#### Changes proposed in this Pull Request
This PR switched from Docker-based JT setup to the standalone version of it. More details in paJDYF-fSK-p2 










#### Testing instructions





* `npm run tube:setup` should run w/o errors, together with using `npm run tube:start` and `npm run tube:stop` 
* `/jurassictube` directory should be created under `/bin` directory, and should be ignored by git
* (optional to check) if downloaded once, the JT installed won't be downloaded once again
* confirm that after multiple subsequent setups, the `config.env` file does not have duplicated keys but instead, contains only rows relevant to the most recent setup

-------------------


**Post merge**
